### ISSUE TITLE
[DatePicker] Navigation inside the popup doesn't work properly when in classic mode under NVDA

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#3361]
**Describe the bug**
Navigation inside the popup when in  "classic" calendar type doesn't work properly under NVDA. Additionally, whether the popup calendar should be one tab stop or prev / next buttons should be included in the tab sequence should be reviewed.

**Comment by dtopuzov** [comment]
Tested with the latest official kendo-angular and latest official NVDA - no issue with keyboard navigation under NVDA (it works exactly as it works without NVDA).
